mage_integrations/mage_integrations/connections/sql/base.py

```
@@ -48,10 +48,13 @@ def get_data_from_query_strings(self, cursor, query_strings):
             self.logger.info(f'{message} started.')
             now1 = datetime.utcnow().timestamp()
 
-            cursor.execute(clean_query(query_string))
-            description = cursor.description
-            if description:
-                data.append(cursor.fetchall())
+            try:
+                cursor.execute(clean_query(query_string))
+                description = cursor.description
+                if description:
+                    data.append(cursor.fetchall())
+            except Exception as err:
+                self.logger.error(f'Error while executing query: {str(err)}')
```

Let's also raise the exception instead of letting it complete.